code_samples/user_management/oauth_google/src/OAuth/GoogleResourceOwnerMapper.php

```diff
@@ -0,0 +1,107 @@
+<?php
+declare(strict_types=1);
```

Comment:
Nitpick: missing new line.

```suggestion

declare(strict_types=1);
```